surveys/2021-contributor-survey/questions.md

```diff
@@ -0,0 +1,495 @@
+> - Make sure none of the major cohort sections are leaving out relevant
+>   questions that are asked of the other major cohorts.
+
+## Collaboration Interview
+
+### Have you contributed or ever considered contributing to the official Rust project (e.g. any repo that is part of the rust-lang Github org)?
+
+- I am an active contributor to the Rust Project. [NEXT](#for-active-contributors)
+- I was an active contributor to the Rust Project but am not any longer. [NEXT](#for-inactive-contributors)
+- I have tried contributing to the Rust Project but stopped before completing any contributions. [NEXT](#for-attempted-contributors)
+- I have wanted to contribute to the Rust Project but have not been able to figure out how. [NEXT](#for-potential-contributors)
+- I have never wanted to contribute to the Rust Project. [NEXT](#for-non-contributors)
+
+> **justification**
+>
+> Fundamental for cohort analysis.
+
+## For active contributors
+
+### How did you get involved in the Rust Project?
+
+Select all that apply:
+
+- I wanted to improve my Rust skills and learn from the experts maintaining Rust.
+- I wanted to implement features that I needed as a user of Rust.
+- I wanted to give back to the Rust community.
+- I wanted to add to my portfolio of open source work for my resume.
+- I was encouraged to contribute by an existing Rust contributor / maintainer.
+- My work assigned open source contributions to me as part of my job.
+- Other: Free form
+
+> **justification**
+>
+> Useful in identifying paths into the project that are successful for future
+> contributors to follow.
+
+### How long have you been contributing to the Rust Project?
+
+- Less than 3 months
+- 3 to 6 months
+- 1 year
+- 2-3 years
+- 3-5 years
+- 5+ years
+
+> **justification**
+>
+> Useful for cohort analysis, the problems faced by contributors may differ
+> substantially for new contributors vs long time maintainers.
+
+### How are you associated with the Rust Project?
+
+- I am a member of a Rust Team.
+- I am a lead of a Rust Team.
+- I am a member of a Rust working group or project group.
+- I am a lead of a Rust working group or project group.
+- I am not a member of a Rust Team, working group, or project group, but I
+  would like to be.
+- I am not a member of a Rust team, working group, or project group, and I have
+  no intention to join one.
+
+> **justification**
+>
+> Cohort analysis. Some of the subsequent questions are only relevant to
+> certain cohorts.
+
+### What kind of work do you do within the Rust Project?
+
+Select all that apply:
+
+- contribution work
+- maintainership work
+- reviews
+- coordination
+- organization
+
+> **justification**
+>
+> Useful for cohort analysis. Problems will likely differ based on the work
+> being done. Those contributing new code will likely face diff problems than
+> those who mainly review or coordinate work.
+
+### What sort of work would you like to be focused on?
```

Comment:
I'm having a hard time understanding exactly what this is asking. Can we word this differently? Perhaps you could clarify what is meant, and I could explain what I find confusing.

Reply:
The goal here is to figure out if people are happy with the work they've been doing or if they're doing it because nobody else is and they know it needs to be done even though it's not something they enjoy. An alternative wording might be "Are you happy with the kind of work you've done so far?" but I think this alternative wording might encourage people to respond with "sure" even though ideally they'd prefer to focus on different work.

As an example, someone who spends the majority of their time reviewing PRs but who would rather be designing and implementing new features.

Reply:
Can we just ask something like: "How much do you agree with this statement: 'The type of work I do in the Rust project is exactly what I'd like to be working on" and then offer options from "Complete agree" to "Completely disagree"